docs/source/contributing/developing.adoc

```diff
@@ -316,6 +316,12 @@ The following options are available:
 
 Tags::
 Use `tags` to ensure that scenarios and features containing at least one of the selected tags are executed.
+To select particular feature you can use its name as a tag, for example: `cmd-config.feature` contains and can be called via `cmd-config` tag.
```

Comment:
For example should be in separate line IMO

Reply:
or just drop the `:`, and it can be part of the sentence. 
```asciidoc
for example, `cmd-config.feature`.....
```

Reply:
Personally, I'd split the information into two sentences. Also, I find the wording of "contains and can be called" vague -- I'm not sure exactly what it means. I'd use something like the following:

```adoc
To select a particular feature, you can use its name as a tag.
For example, `cmd-config.feature` can be called via the `cmd-config` tag.
```